mne/inverse_sparse/mxne_inverse.py

```diff
@@ -364,15 +375,22 @@ def mixed_norm(evoked, forward, noise_cov, alpha, loose='auto', depth=0.8,
     .. footbibliography::
     """
     from scipy import linalg
-    if not (0. <= alpha < 100.):
+    if type(alpha) == int and not (0. <= alpha < 100.):
         raise ValueError('alpha must be in [0, 100). '
                          'Got alpha = %s' % alpha)
+    elif type(alpha) == str and alpha != "sure":
```

Comment:
```suggestion
    elif isinstance(alpha, str) and alpha != "sure":
```

using type(xxx) == yyy is evil :)